desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -98,8 +98,8 @@ private int brighter(final int rgb, final int percent) {
 
     private static final WidgetScreenResolutionAdjuster adjuster = WidgetScreenResolutionAdjuster.get();
     public static final int ICON_SIZE_LARGE = adjuster.adjust(32);
-    public static final int ICON_SIZE_MEDIUM = adjuster.adjust(22);
-    public static final int ICON_SIZE_SMALL = adjuster.adjust(16);
+    public static final int ICON_SIZE_MEDIUM = adjuster.adjust(24);
+    public static final int ICON_SIZE_SMALL = adjuster.adjust(18);
```

Comment:
This seems unrelated, and a bit dangerous to just adjust all icon sizes like this?

Reply:
Hmm yes I admit this one does not fit in... I'll revert.
The background is that the new icon for machine learning components is very hard to distinguish when overlayed with the "folder" icon. But maybe that just means that a better icon needs to be built.